app/controllers/documents_controller.rb

```diff
@@ -20,7 +20,6 @@ def index
 
   def new
     @document = Document.new
-    verify_org
```

Comment:
verify_org verifies the organization with a documents.organization_id. this dose not work for a new document.